crates/tests/databases-tests/src/postgres/snapshots/databases_tests__postgres__mutation_tests__basic__insert_artist_album.snap

```diff
@@ -2,40 +2,51 @@
 source: crates/tests/databases-tests/src/postgres/mutation_tests.rs
 expression: result
 ---
-{
-  "operation_results": [
-    {
-      "affected_rows": 1,
-      "returning": [
-        {
-          "__value": [
-            {
-              "artist_id": 276,
-              "name": "Olympians"
-            }
-          ]
-        }
-      ]
-    },
+[
```

Comment:
we modified the existing test to also run a select after the mutation to make sure that the insert worked. We put the two (separate) requests in a tuple, which is why this is now an array.